src/channel.ts

```diff
@@ -352,23 +352,20 @@ export class Channel<
   }
 
   /**
-   * search - Query Members
+   * queryMembers - Query Members
    *
    * @param {UserFilters<UserType>}  filterConditions object MongoDB style filters
-   * @param {UserSort<UserType>} [sort] Sort options, for instance {created_at: -1}
+   * @param {UserSort<UserType> | UserSort<UserType>[]} [sort] Sort options, for instance {created_at: -1}.
+   * When using multiple fields, make sure you use array of objects to guarantee field order, for instance [{last_active: -1}, {created_at: 1}]
    * @param {{ limit?: number; offset?: number }} [options] Option object, {limit: 10, offset:10}
    *
-   * @return {Promise<ChannelMemberAPIResponse<UserType>>} search members response
+   * @return {Promise<ChannelMemberAPIResponse<UserType>>} Query Members response
    */
   async queryMembers(
     filterConditions: UserFilters<UserType>,
-    sort: UserSort<UserType> = {},
+    sort: UserSort<UserType> | UserSort<UserType>[] = {},
```

Comment:
we should probably make the `UserSort<UserType>[]` a mutually exclusive type so that each array entry object has only one attribute inside. @nhannah is it possible?

Reply:
I tried this with union type like this:

```typescript
export type ChannelSort = {created_at: AscDesc} | {updated_at: AscDesc}
```

but it seemed ugly to me. Also, type parameters of `Sort` made it so bad so I couldn't come up with proper `buildSort` input type which will work for everyone. As I said, would appreciate any help in this area